# The Effect Size Files — Week 6
**Claim:** BrainBoost Gum Co.

## Claim

BrainBoost Gum Co. states that chewing their nootropic gum for 10 minutes before an exam improves memory recall scores more than chewing regular mint gum.

- **Outcome measured:** memory recall score (continuous)
- **Comparison:** nootropic gum group vs. regular mint gum group

## Hypotheses

- **Null (H₀):** The mean memory recall score of students who chew nootropic gum 10 minutes before an exam is equal to the mean score of students who chew regular mint gum.
- **Alternative (H₁):** The mean memory recall score of students who chew nootropic gum 10 minutes before an exam is greater than the mean score of students who chew regular mint gum.

This is a **one-sided** alternative because BrainBoost's claim is directional (gum *improves* recall), and there is no theoretical mechanism by which the nootropic ingredients would harm recall relative to mint.

## Statistical Test

**Independent samples t-test, one-sided.**

Justification:
- Two independent groups (nootropic vs. mint, no overlap of participants)
- Continuous outcome (memory recall score)
- Comparing group means
- Directional hypothesis (one-sided)

## Assumptions

1. **Approximate normality** — Memory recall scores are assumed roughly normally distributed within each group. With a sufficient sample size (n ≥ 30 per group), the Central Limit Theorem makes the t-test robust to moderate departures from normality.
2. **Equal variances** — Both groups are randomly assigned from the same student population, so there is no theoretical reason to expect one group to have greater spread than the other. If violated in practice, Welch's t-test is a direct substitute.
3. **Independence of observations** — Random assignment to groups and individual (non-group) test administration ensure each student's score is independent of every other's.

## Literature Overview

Prior research on gum chewing and memory recall is mixed. Wilkinson et al. (2002) first demonstrated that chewing gum improves both short-term and long-term word recall in healthy volunteers. Davidson (2011) extended this to herbal-caffeinated gum, showing memory improvements over both regular gum and no-gum controls. However, subsequent studies (Tucha et al., 2004; Miles & Johnson, 2007) have failed to replicate these effects.

Given this inconsistency, and because no reviewed study reports a Cohen's d directly comparable to our two-group design, we plan conservatively for a **medium effect size (Cohen's d = 0.5)** per Cohen's (1988) conventions. A conservative estimate reduces the risk of running an underpowered study if the true effect is smaller than the most optimistic findings suggest.

## References

Cohen, J. (1988). *Statistical power analysis for the behavioral sciences* (2nd ed.). Hillsdale, NJ: Lawrence Erlbaum Associates.

Davidson, M. G. (2011). Herbal-caffeinated chewing gum, but not bubble gum, improves aspects of memory. *Appetite*, 57(1), 303–307. https://doi.org/10.1016/j.appet.2011.04.019

Miles, C., & Johnson, A. J. (2007). Chewing gum and context-dependent memory effects: A re-examination. *Appetite*, 48(2), 154–158. https://doi.org/10.1016/j.appet.2006.07.082

Tucha, O., Mecklinger, L., Maier, K., Hammerl, M., & Lange, K. W. (2004). Chewing gum differentially affects aspects of attention in healthy subjects. *Appetite*, 42(3), 327–329. https://doi.org/10.1016/j.appet.2004.01.003

Wilkinson, L., Scholey, A., & Wesnes, K. (2002). Chewing gum selectively improves aspects of memory in healthy volunteers. *Appetite*, 38(3), 235–236. https://doi.org/10.1006/appe.2002.0473

In [5]:
from statsmodels.stats.power import TTestIndPower

In [15]:
# Parameters
effect_size = 0.5
alpha = 0.05
power = 0.8

In [16]:
# Power analysis
analysis = TTestIndPower()
n_size = analysis.solve_power(
    effect_size=effect_size, 
    alpha=alpha, 
    power=power, 
    alternative='larger')


In [17]:
print(n_size)

50.15078407854097


## Interpretation

The power analysis returned **n ≈ 50.15 per group**, which rounds up to **51 participants per group**, or **102 students total**.

This means: if BrainBoost's nootropic gum truly produces a medium effect on memory recall (Cohen's d = 0.5), recruiting 51 students into the nootropic condition and 51 students into the mint-gum control condition gives us an **80% probability of detecting that effect** as statistically significant at the α = 0.05 level (one-sided).

A few practical implications:

- **Rounding up matters.** 50.15 rounded down to 50 would leave the study slightly underpowered. Always round up on sample size calculations.
- **This is a minimum, not a target.** 102 is the floor. Recruiting more buffers against dropouts, exclusions due to protocol violations (e.g., students who forget to chew the gum), and any over-optimism in our effect size assumption.

In short, **BrainBoost should plan to recruit at least 102 students (51 per group)**